talk/morelanguage/move.tex

```diff
@@ -93,12 +95,12 @@
   \frametitlecpp[11]{Move semantics}
   \begin{block}{The idea}
     \begin{itemize}
-      \item a new type of reference : rvalue references
+      \item a new type of reference : rvalue reference
       \begin{itemize}
       \item used for move semantic
-      \item denoted by \&\&
+      \item denoted by \mintinline{cpp}{&&}
       \end{itemize}
-      \item 2 new members in every class, with move semantic :
+      \item 2 new members in every class, with move semantic:
```

Comment:
I have seen this in many places in the presentation. Is there a reason the colon is not attached to the last word?

Reply:
That's how you typeset in French texts. I think it's an oversight when this makes it into the text.

Reply:
Oh, I did not know. Thx! I will revert those changes then!